Tasks/Common/webdeployment-common-v2/packageUtility.ts

```diff
@@ -29,12 +29,36 @@ export class Package {
     constructor(packagePath: string) {
         this._path = PackageUtility.getPackagePath(packagePath);
         this._isMSBuildPackage = undefined;
+        let workingDirectory = tl.getVariable("system.defaultworkingdirectory");
+        try {
+            if (workingDirectory && this._path.indexOf(workingDirectory) == 0) {
```

Comment:
also how does this work with the multi-stage yml pipelines

Reply:
I will have to check this scenario.